### ISSUE TITLE
"Search for Lyrics (manually)" Window - #1 Allow Dark Mode, #2 Automatically sort search results with Timestamped at the top

### ISSUE THREAD

**Issue by ngs428** [jacquesh/foo_openlyrics#398]
This window does not support Dark Mode.  It is the only one I can find in OpenLyrics which is that way.  Can that be changed?  

What right clicking on a file and selecting "Open Lyrics" -> "Search for Lyrics (manually)" the lyrics don't appear in any specific order.  One then has to click on the "Is Timestamped" header twice to move the timestamped lyrics to the top.  Would be quicker to have the search results already sort  by this field with the timestamped lyrics at the top.  

Initial Result sorting (and note lack of Dark Mode):
![Initial Results](https://github.com/user-attachments/assets/e27854ea-213a-4991-92e9-b4bf1a33490b)

Preferred Search order (sorted with timestamped at top):
![Sorted](https://github.com/user-attachments/assets/6aa06663-6f70-4729-98fe-8203adab6b07)

**Comment by jacquesh** [comment]
Results are displayed by default in the order that they were received. This is intentional so that entries in the list do not move around while the search is ongoing. Imposing other other default sort order would take that way. Is that worth it to save 2 extra clicks on the occasion that you do a manual search (which we could probably also reduce to 1 extra click by reversing the default order of the timestamp column)? Eh, maybe? I'm not immediately convinced.

Also not everybody specifically wants timestamped lyrics.

**Comment by ngs428** [comment]
Makes sense.  Certainly different use cases for many different users.  If you could change it to the first click of the “is timestamped” header makes the time stamped at the top, that would be a good move. Save some time, I would appreciate that.  I assume more users use timestamped than not. 

As I mentioned in #400 this is my and many people’s main method of searching.  So anything to speed it up, like only showing sources you want to search and with 1 click having timestamped at the top would be great.  Thank you.

**Comment by drelephant** [comment]
How about two checkboxes, one to enable showing timestamped and one to show non-timestamped files?

Personally I'm usually searching for timestamped lyrics. If those aren't available then I'll settle for non-timestamped.

**Comment by jacquesh** [comment]
I think that's a bit of a slippery slope. If we have a checkbox for timestamped, why not also for artist/album/title/source? I expect that would quickly get out of hand. Sorting is already consistently available and just sorting the interesting results to the top is largely equivalent (in the sense that you see all the results you want). Sorting also makes the fallback to non-timestamped a tiny bit simpler (because you don't have to click anything, you just keep scrolling).

**Comment by drelephant** [comment]
Well if it was up to me, I'd just remember the sort order and add the new incoming files in sorted order as they come in.

How about a tickbox to preserve sorting for new items?

Then people wouldn't have to click the sort header every time, with usually multiple clicks as the results populate.

I'm just spitballing a solution so I can avoid the clicks every time, and it seems I'm not the only one who would like this.
